test/e2e-shared-tests/mcd_config_drift.go

```diff
@@ -218,6 +231,62 @@ func (c configDriftTest) Run(t *testing.T) {
 				})
 			},
 		},
+		{
```

Comment:
Just so I understand what this new test does:
1. We mutate one of the files to cause a degradation and we capture the degradation reason.
2. Next, we recover from the degradation.
3. We mutate a different file to cause a degradation.
4. We mutate the original file a second time to cause degradation.
5. We compare the third degradation (from the same file as the first) to ensure that this is the degradation reason we see instead of the second one, correct?

Reply:
Yes, that is what the test is intended to do. I will add that snippet, thanks!

Reply:
You might want to add some additional assertions to both the second and third filesystem mutations so that it's even more obvious what's going on.